scripts/next-release.sh

```diff
@@ -7,21 +7,38 @@
 # https://docs.google.com/document/d/11cw-7dAp93JmasITNSNCtx31xrQsNB1L2OoxVE6zrTc/edit#bookmark=id.ufwe0bqp83z1
 set -eu
 
-if ! command -v gdate &>/dev/null; then
-  echo "command not found: gdate"
-  echo "The command gdate is required to compute the next version number"
-  echo "To fix this problem, run:\n  brew install coreutils"
-  exit 1
+if [[ "$(uname)" == "Darwin" ]]; then
+  if ! command -v gdate &>/dev/null; then
+    echo "Command not found: gdate"
+    echo "The command gdate is required to compute the next version number"
+    echo "To fix this problem, run:\n  brew install coreutils"
+    exit 1
+  fi
+  date_program() {
+    gdate "$@"
+  }
+else
+  if ! command -v date &>/dev/null; then
+    echo "Command not found: date"
+    exit 1
+  fi
+  date_program() {
+    date "$@"
+  }
 fi
 
+if ! command -v gh &>/dev/null; then
+  echo "Command not found: gh"
```

Comment:
would it make sense to add a similar message like for gdate? I mean smth like
```
    echo "The command gh is required to compute the next version number"
    echo "To fix this problem, run:\n  brew install gh"
```

Reply:
Naah, I'm just merging - looks like it is going to be removed soon. But thanks for comment!